openmetadata-service/src/main/java/org/openmetadata/service/migration/mysql/v170/Migration.java

```diff
@@ -28,6 +29,9 @@ public void runDataMigration() {
     runLineageMigrationForNullColumn(handle);
     runLineageMigrationForNonNullColumn(handle);
 
+    // DI
```

Comment:
Do we need this comment?

Reply:
Yes I added this because someone did a mistake while resolving conflict so just to make it easier... DI is data insights 😅